orchestration/src/test/java/com/sap/ai/sdk/orchestration/LLMModuleResultDeserializerTest.java

```diff
@@ -0,0 +1,95 @@
+package com.sap.ai.sdk.orchestration;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sap.ai.sdk.orchestration.client.model.LLMModuleResult;
+import com.sap.ai.sdk.orchestration.client.model.LLMModuleResultStreaming;
+import com.sap.ai.sdk.orchestration.client.model.LLMModuleResultSynchronous;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.Test;
+
+class LLMModuleResultDeserializerTest {
```

Comment:
(Comment/Minor)

Test class looks good!
Only wondering whether we could remove a few new-lines. Or remove the comments.